runners/direct-java/src/main/java/org/apache/beam/runners/direct/ForwardingPTransform.java

```diff
@@ -57,6 +57,6 @@ public String getName() {
 
   @Override
   public void populateDisplayData(DisplayData.Builder builder) {
-    delegate().populateDisplayData(builder);
+    builder.delegate(delegate());
```

Comment:
Why couldn't this stay `delegate().populateDisplayData(builder)`? Will this have any side-effects on what information is actually produced?